### ISSUE TITLE
Split AWX collections into its own repo from AWX

### ISSUE THREAD

**Issue by aheath1992** [ansible/awx#14438]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx/issues) for duplicates.
- [X] I understand that AWX is open source software provided for free and that I might not receive a timely response.

### Feature type

New Feature

### Feature Summary

Split AWX collection into its own repo under the ansible namespace. As a package maintainer for the [AWX collection rpm](https://src.fedoraproject.org/rpms/ansible-collection-awx-awx) we try to follow the release cycle of AWX, however in the latest release the only thing that was changed for the collection was the license.

> Fix collection metadata license to match intent by @AlanCoding in https://github.com/ansible/awx/pull/14404

I believe separating out the collection to its own repo will allow it to have its own lifecycle not tied to the AWX product release.

### Select the relevant components

- [ ] UI
- [ ] API
- [ ] Docs
- [X] Collection
- [ ] CLI
- [ ] Other

### Steps to reproduce

Separate awx-collection to it's own repo under the ansible namespace.

### Current results

awx-collection is under the AWX namespace their for the RPM release it tied to the AWX release when nothing may have changed with the collection.

### Sugested feature result

A separate repo that houses the AWX collection that has its own release cycle and the RPM packaging aligns with rather then the AWX repo

**Comment by thedoubl3j** [comment]
Hey @aheath1992 what is the exact issue? Is it that you are release redundant awx RPMs? What could make your life easier for the RPM management aside from cutting it out of the repo?

**Comment by aheath1992** [comment]
@thedoubl3j Yes, to both your points We would release non redundant RPMs and Galaxy releases, and it would allow the awx collection to follow the [Ansible Collection Packaging Guidelines](https://docs.fedoraproject.org/en-US/packaging-guidelines/Ansible_collections/) a bit better.

Over all their is no issue just a quality of life improvement more than anything, I have also opened a discussion thread over on the [Ansible forum](https://forum.ansible.com/t/reason-for-awx-collection-locaton/617) to request more feedback as well.
